content/storage/_index.md

```diff
@@ -106,6 +106,12 @@ You can't use Docker CLI commands to directly manage bind mounts.
 > non-Docker processes on the host system.
 { .important }
 
+> **Tip**
+>
+> Working with large repositories or monorepos, or with virtual file systems that are no longer scaling with your codebase?
+> Checkout [Synchronized File Shares](../desktop/synchronized-file-sharing.md) which provides fast and flexible host-to-VM file sharing by replacing bind mounts with synchronized filesystem caches.
```

Comment:
question: We say "replacing bind mounts" here and in a few other places, but I don't think this is the correct way to phrase this. You'd still use a bind mount to get the files into the container, correct? So it doesn't replace bind mounts, but it makes bind mounts faster by adding a file synchronization layer between host and VM, and then bind mounts from VM to container I guess?

```suggestion
> Check out [Synchronized File Shares](../desktop/synchronized-file-sharing.md) which provides fast and flexible host-to-VM file sharing by replacing bind mounts with synchronized filesystem caches.
```

Reply:
Also, should this be changed?

"Synchronized File Shares" -> "Synchronized file sharing"

Reply:
> Nope this is correct, and also how it is phrased on the Mutagen docs site https://mutagen.io/documentation/docker-desktop-extension

How do you get those filesystem caches into the containers? You would use `--mount=type=bind` or `-v <src>:<dest>` -- a bind mount. Bind mounts aren't Docker Desktop implementations, they're a feature of the Docker Engine. It's my understanding that this feature changes how files move between the host and the container, but at the end of the day you're still using a bind mount when you create the container. That's why I don't think "replacing bind mounts" is a good choice of terminology.

cc @thaJeztah

Reply:
I agree with @dvdksn, and perhaps the verbage on the Mutagen extension documentation isn't ideal.  At the end of the day it's a nuanced technical distinction that we should hide from users (without being incorrect in our description).

How about a description like "Synchronized File Shares is an alternative file sharing mechanism that provides fast and flexible host-to-VM file sharing that enhances bind mount performance using synchronized filesystem caches."

Reply:
Yes, I like that approach. Effectively, it's providing the existing functionality, but making it _better_ (same UX, better experience)